assets/js/components/setup/compatibility-checks.js

```diff
@@ -122,35 +122,39 @@ export default class CompatibilityChecks extends Component {
 
 		if ( ! installed && installURL ) {
 			return {
-				labelHTML: __( 'Install<span class="screen-reader-text"> the helper plugin</span>', 'google-site-kit' ),
+				ariaLabel: __( 'Install the helper plugin', 'google-site-kit' ),
+				label: __( 'Install', 'google-site-kit' ),
```

Comment:
I mentioned this before but why not use `children` instead and spread into the `Link` below since these would all be props now? Not a big deal, just curious if there is a reason to do it this way instead.

Reply:
@aaemnnosttv Makes sense :) I didn't know about the children props :(